test/e2e/test-scripts/view-auction.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
consider using Node.js scripts instead. Especially for tests. LTS has a built-in test runner https://nodejs.org/docs/latest-v20.x/api/test.html. Shell commands can run with https://nodejs.org/docs/latest-v20.x/api/child_process.html#child_processexecsynccommand-options. It even has JSON parsing ;-)